mkosi/resources/man/mkosi.1.md

```diff
@@ -1208,6 +1208,11 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
     `secure-boot-enroll manual` in it. Auto enrollment is not supported on
     systemd versions older than v252. Defaults to `yes`.
 
+`SecureBootAutoEnrollCertificates=`, `--secure-boot-auto-enroll-certificates=`
+:   If `SecureBootAutoEnroll=` is enabled, appends additional certificates to
+    UEFI db. Expects comma-separated list of paths to PEM formatted X.509
+    certificates. Only implemented for images built on OBS for now.
```

Comment:
For `SecureBootAutoEnroll` we have a pre-257 and a post-257 code path. The pre-257 codepath actually seems more amenable to adding this quickly, the post-257 might need some work in bootctl, since, I've only looked at the man page, it seems to require cert and key.

Generally I'd prefer to have this option hidden and undocumented as long as it cannot be used outside of OBS.

Reply:
I don't think it makes sense to spend time to add support for something new to a legacy code path, so I've removed it from the manpage for now

Reply:
Sorry if I was unclear, I didn't want to imply that it should be added to the legacy codepath, I'm totally fine with adding it to the post-257 codepath only. I just wanted to say that it seems to need a bit more work outside of mkosi. I do think this could be useful and you find the time to get around to it.

That being said, while that work is undergoing, I prefer this option not to be user-visible.